app/dashboard/dashboard.component.e2e-spec.ts

```diff
@@ -0,0 +1,21 @@
+import { browser, by, element } from 'protractor'
+
+describe('Dashboard', () => {
+
+  beforeEach(() => {
+    browser.get('/dashboard');
+  });
+
+
+  it('should show dashboard', () => {
+    browser.wait(function () {
+      return element(by.css('hip-dashboard md-card h2')).isPresent();
```

Comment:
In the current code the title is an `h1` element.
But it's enough to select with `hip-dashboard md-card` or even `md-card`